tests/integration/steps.go

```diff
@@ -23,7 +23,9 @@ var happyPathSteps = concatSteps(
 	stepsRejectEquivocationProposal("consu", 2),   // prop to tombstone bob is rejected
 	stepsDoubleSignOnProviderAndConsumer("consu"), // carol double signs on provider, bob double signs on consumer
 	stepsSubmitEquivocationProposal("consu", 2),   // now prop to tombstone bob is submitted and accepted
-	stepsStopChain("consu", 3),
+	stepsStartHermes(),
+	stepsConsumerRemovalPropNotPassing("consu", 3), // submit removal prop but vote no on it - chain should stay
```

Comment:
Irrelevant for now but perhaps future we move away from hard coding the proposal number and pass in something that can auto increment.

Reply:
Nope, not how this is supposed to work, but we can talk about that later at the retreat with @p-offtermatt